docker-compose.yml

```diff
@@ -73,7 +74,7 @@ services:
         cp /broker-certs/client.crt /etc/iib/messaging.crt &&
         cp /broker-certs/client.key /etc/iib/messaging.key &&
         cp /broker-certs/ca.crt /etc/iib/messaging-ca.crt &&
-        pip3 uninstall -y iib &&
+        (pip3 uninstall -y iib || rm -rf /usr/local/lib/python3.8/site-packages/iib/) &&
```

Comment:
This is not ideal - problem is when the uninstall takes place it does not delete the folder completly. `exceptions.py` is still present.

Reply:
Excellent!

Unfortunately I'm still not able to reproduce this issue.

I even booted a fresh VM with Fedora 34, cloned the project and it build normally with worker up.

Anyway, it makes sense to do this, it ensures the complete removal off IIB when rebuilding.